AK/MemoryStream.h

```diff
@@ -34,6 +34,8 @@ class FixedMemoryStream final : public SeekableStream {
     size_t offset() const;
     size_t remaining() const;
 
+    bool is_writing_enabled() const { return m_writing_enabled; }
```

Comment:
This still isn't used anywhere.

Reply:
It was used previously, but now we don't need it anymore.